webapp/src/constants/scopeCategories.ts

```diff
@@ -0,0 +1,50 @@
+import { expandScope, Scope } from 'tg.component/permissions/permissionHelper';
+
+export function getScopeCategories(t: (key: string) => string) {
+  const scopeCategories = [
+    {
+      name: t('scope_category_project_management'),
```

Comment:
:eyes:
```suggestion
      // @tolgee-key scope_category_project_management
      name: t('scope_category_project_management'),
```

Reply:
Why?

Reply:
The `t` doesn't come from a direct import, so the CLI would be unable to detect these. Adding annotations would make sure the CLI sees these and extracts them

Reply:
Oh. Right!